### PR TITLE
High voltage design for when a course has no assessments

### PR DESCRIPTION
**Pull request by mborsare**
**small screen**
screen shot 2017-05-16 at 5 08 28 pm

**big screen**
hover

### CODE REVIEW

File: app/views/pages/course-no-classes.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <h2 class="headline-poster headline-poster-copy narrow-content">
+    Aerospace Engineering doesn’t have any classes yet
+  </h2>
+</div>
+
+<%= link_to "Add a class", "#!", class: "headline-poster headline-poster-action" %>
+
+<script type="text/javascript">
+  $(function() {
+    $(".headline-poster-action").mouseenter(function(){
```
Comment (ryancoughlin) [inline app/views/pages/course-no-classes.html.erb]: Space between parens `() {`

File: app/views/pages/course-no-classes.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      $("body").addClass('headline-poster-takeover');
+    });
+    $(".headline-poster-action").mouseleave(function(){
```
Comment (ryancoughlin) [inline app/views/pages/course-no-classes.html.erb]: You won't notice this much in this case above, but it's best to cache selectors (this is so lightweight though):

`var $headlinePosterAction = $(".headline-poster-action");`

File: config/locales/manage_assessments.en.yml [hunk 3]
```diff
@@ -65,6 +65,9 @@ en:
         description: Description
         heading: Outcomes for %{name}
         label: Label
-      show:
+      show: &courses_show
```
Comment (margaretoti) [inline config/locales/manage_assessments.en.yml]: Out of curiosity, why does `show:` need to be aliased to `courses_show`?

**Comment by mborsare** [comment]
LGTM!

**Pull request closed without merging**